pyproject.toml

```diff
@@ -9,20 +9,20 @@ authors = [{ name = "The Marabou Development Team" }]
 description = "A package wrapping the Marabou Neural Network Verification tool."
 readme = "README.md"
 license = { file = "COPYING" }
-requires-python = ">=3.7,<3.12"
+requires-python = ">=3.8,<3.12"
 
 [project.optional-dependencies]
 test = [
   # NOTE: These dependencies must be kept in-sync with the legacy requirements file 'maraboupy/test_requirements.txt'
   "numpy>=1.21.0,<2",
-  "onnx>=1.12.0,<2",
+  # NOTE: Marabou requires onnx.reference which was introduced in onnx==1.15.0
```

Comment:
Apologies, I wasn't clear enough before. I meant that we should add a comment here saying: "The ONNX version must be kept in sync with `tools/download_onnx.sh`"

Reply:
Ah I see. Added the note.